src/data/set/pointwise/finite.lean

```diff
@@ -82,6 +82,23 @@ open set
 
 namespace group
 
+lemma card_pow_eq_card_pow_card_univ_aux {f : ℕ → ℕ} (h₁ : monotone f) {B : ℕ} (h₂ : ∀ n, f n ≤ B)
```

Comment:
A nit, but this move looks like it could be its own PR

Reply:
Heather was the one moving the main lemma away from the auxiliary one in the previous split, but I agree this could be promoted to a normal lemma, rather than just an auxiliary one.

Reply:
Presumably the lemma being left behind was an accident, and @hrmacbeth only moved the lemmas that actually used the imports that were changed.

Reply:
Moved to `data.nat.order.basic` for now.